### PR TITLE
Ben/43/roles page

### LINKED ISSUE #43: Roles page start to finish 🏃‍♂️
Finish up the roles page sections and header. Reference the Figma mockups here:
https://www.figma.com/file/aAThxeD86vXciqJRTmDLjyQG/BitsOfGood-web-redesign?node-id=207%3A0

### PR DESCRIPTION
**Pull request by ghost**
Resolve #43 

It's finally here. And it's beautiful.

### CODE REVIEW

File: src/components/about-us/roles/Department.svelte [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    .content-container h2 {
+      position: absolute;
+      transform: rotate(0.25turn);
+      transform-origin: top right;
+      width: 400px;
+    }
+  }
+</style>
+
+<section class:rightAlign id={hash}>
```
Comment (chrisjobando) [inline src/components/about-us/roles/Department.svelte:117]: why do you use a colon operator here

Comment (ghost) [reply src/components/about-us/roles/Department.svelte:117]: @chrisjobando Svelte magic! It's shorthand for something like this: `class={rightAlign ? 'rightAlign' : ''}`

**Review by chrisjobando — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -17,7 +17,8 @@
     "compression": "^1.7.1",
     "netlify-lambda": "^1.6.3",
     "polka": "^0.5.0",
-    "sirv": "^0.4.0"
+    "sirv": "^0.4.0",
```
Comment (karpawich) [inline package.json]: Bad package. bad

File: src/components/about-us/roles/RoleDropdown.svelte [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      <h3>{name}</h3>
+    </button>
+    {#if applicationOpen}
+      <a class="apply-link" href={applyLink}>Apply Now</a>
+    {/if}
+
+  </div>
+  {#if open}
+    <div class="description" transition:fly={{ y: -20, duration: 200 }}>
+      {@html description.html}
```
Comment (karpawich) [inline src/components/about-us/roles/RoleDropdown.svelte:89]: are their any preventative measures for XSS here?

**Review by karpawich — APPROVED** [review]
- updated sapper to resolve critical NPM vulnerability

File: src/components/about-us/roles/RoleDropdown.svelte (hunk 3 above)
Comment (ghost) [reply src/components/about-us/roles/RoleDropdown.svelte:89]: Svelte is smart enough to handle that

**Pull request merged**